internal/pkg/skuba/skuba/configmap.go

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2019 SUSE LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package skuba
+
+import (
+	"github.com/pkg/errors"
+
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+
+	"github.com/SUSE/skuba/internal/pkg/skuba/kubernetes"
+)
+
+const (
+	ConfigMapName             = "skuba-config"
+	SkubaConfigurationKeyName = "SkubaConfiguration"
+)
+
+type SkubaConfiguration struct {
+	AddonsVersion kubernetes.AddonsVersion
+}
+
+func GetSkubaConfiguration() (*SkubaConfiguration, error) {
```

Comment:
Naming as GetConfiguration is better.
Then use it, is `skuba.GetConfiguration`.